extension/src/background.js

```diff
@@ -117,7 +117,8 @@ function getDelayMs(/*url*/) {
     return 10 * 1000;
 }
 
-const LOCAL_TAG = 'local';
+//hmm having a space in a tag shouldn't cause an error but it does
+const LOCAL_TAG = 'this_browser_history';
```

Comment:
Hmm, not sure about this.. I think for now I'd rather prefer this to be shorter (because it's displayed by default and might be quite spammy).
If you want to configure it for yourself, you can use "source map" in the settings. Would that work for you?

![image](https://user-images.githubusercontent.com/291333/97372354-5d605280-18ab-11eb-9e9f-919a7edcac4a.png)